pom.xml

```diff
@@ -179,7 +179,7 @@
     <maven.compiler.target>${mojo.java.target}</maven.compiler.target>
     <maven.compiler.showDeprecation>true</maven.compiler.showDeprecation>
 
-    <minimalJavaBuildVersion>${mojo.java.target}</minimalJavaBuildVersion>
+    <minimalJavaBuildVersion>11</minimalJavaBuildVersion>
```

Comment:
please note njord is requiring 11 but this doesn't change compiler target

Reply:
ups I missed it ... Will it means that we can not use a JDK 8 for build anymore ...

Not sounds good ...

Reply:
> ups I missed it ... Will it means that we can not use a JDK 8 for build anymore ...
> 
> Not sounds good ...

target still 8 so it's perfectly fine.
using 11 in 2025 shouldn't be a problem :)

Reply:
Maybe simply test only with 17 and 21+?

Reply:
Jdk 8 was here to make sure that we keep compatibility (compile and runtime tests). Almost nowhere we are using 11 - it was skipped to save on CI

Reply:
If we agree that we can skip runtime tests with JDK8 it is ok for me.
I would not like to introduce things like toolchains for such tests ... it will complicate project more.

Reply:
we will have:
- https://github.com/maveniverse/njord/pull/98

Reply:
well maybe in 2025, Java 11 as a minimum could make sense :)

Reply:
So if we decide to require 11 for new plugin releases, it effectively limits updates in our other Apache plugins.

Reply:
but ok fair enough to have java 8 :) 
I will change this

Reply:
pending https://github.com/maveniverse/njord/pull/98